vnext/Desktop/React.Windows.Desktop.vcxproj.filters

```diff
@@ -81,9 +81,6 @@
     <Midl Include="ABI\NativeTracing.idl">
       <Filter>ABI</Filter>
     </Midl>
-    <Midl Include="..\Microsoft.ReactNative\ReactInstance.idl">
-      <Filter>ABI</Filter>
-    </Midl>
```

Comment:
Is it an intentional change?

Reply:
Nope. IDE-generated, and innocuous to the actual project structure (note the `.filters` extension).